packages/storybook/stories/modal-dialog.mdx

```diff
@@ -1,22 +1,62 @@
-import { ArgTypes, Meta, Stories } from '@storybook/blocks';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
 import { ModalDialog } from "@equisoft/design-elements-react";
 import * as ModalDialogStories from './modal-dialog.stories';
 
+# Modal
+
+## Table of contents
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+5. [Accessibility](#accessibility)
+
 <Meta of={ModalDialogStories} />
 
-# Modal Dialog
+## Definition
+A modal is an overlay that requires the user to interact with it and is designed to elicit a response from
+the user (dialog). Modals inform users about critical information and require users to make decisions. Within apps,
+on the web and even on mobile, modals are increasingly used to direct the user’s attention to a specific task, without
+taking them away from the context of the current screen [1].
```

Comment:
ça vient quand même changer le contexte étant donné que l'utilisateur n'a plus accès au main content.

> a change of context occurs when there is a user interface change that impacts the focus, selection, or content of the page without informing the user beforehand.
When a modal opens, it can shift the user's focus away from the main content of the page and onto the modal itself